partition/utils.go

```diff
@@ -50,3 +53,20 @@ func runCommandImpl(args ...string) (string, error) {
 
 	return stdout.String(), err
 }
+
+func find(root, fn string) (string, error) {
```

Comment:
See earlier note. I don't think we want this function. We shouldn't be guessing the location of a bootloader config (!).